lib/evss/letters/service.rb

```diff
@@ -3,24 +3,31 @@
 require 'common/client/base'
 require 'common/exceptions/internal/record_not_found'
 require 'common/exceptions/external/gateway_timeout'
+require 'common/client/concerns/monitoring'
 
 module EVSS
   module Letters
     class Service < EVSS::Service
+      include Common::Client::Monitoring
+
       configuration EVSS::Letters::Configuration
 
       def get_letters
         with_monitoring do
           raw_response = perform(:get, '')
           EVSS::Letters::LettersResponse.new(raw_response.status, raw_response)
         end
+      rescue StandardError => e
+        handle_error(e)
```

Comment:
Nitpick warning.  Isn't this the preferable syntax?

```
begin
  do_something()
rescue => e
  # This is the same as rescuing StandardError
end
```
([source](http://blog.honeybadger.io/ruby-exception-vs-standarderror-whats-the-difference/))

Reply:
I agree! New rubocop wants it like above but I think we should exclude that rule

Reply:
Interesting.  Seems fine to leave it as is for this PR but I'm comfortable whatever.